components/SendMoneyToCollectiveBtn.js

```diff
@@ -92,22 +104,22 @@ class SendMoneyToCollectiveBtn extends React.Component {
   }
 }
 
-const paymentMethodsQuery = gql`
+const paymentMethodsQuery = gqlV2/* GraphQL */ `
   query SendMoneyToCollectivePaymentMethods($slug: String) {
-    Collective(slug: $slug) {
+    account(slug: $slug) {
       id
-      paymentMethods(service: "opencollective") {
+      paymentMethods(service: [OPENCOLLECTIVE], type: ["collective"]) {
```

Comment:
Normally, you should be able to write. 

`service: OPENCOLLECTIVE, type: "collective"`

Would looks simpler.

Reply:
@znarf : Thanks done. Sorry liked to keep it the array notation at first since it makes it clear what the api args expect. But yes this is simpler. 👍🏽